mobile/lib/services/location_service.dart

```diff
@@ -161,22 +167,7 @@ class LocationService {
     position ??= await getCurrentPosition();
 
     if (position == null) {
-      final geoCoordinates = await AirqoApiClient().getLocation();
-      if (!geoCoordinates.keys.contains('latitude') ||
```

Comment:
Hi @NoahNsimbe , is this where we disable the usage of IP address? If not, please point me to that place? Thanks!